origin.html

```diff
@@ -16,80 +16,189 @@
+            </div>
+            <div class="firstsection-column2">
+                <ul class="socialmedia">
+                    <li><a href=""><img src="./images/github.svg" alt="githublogo"></a></li>
+                    <li><a href=""><img src="./images/Linkedinicon.svg" alt=""></a></li>
+                    <li><a href=""><img src="./images/angellist.svg" alt="angellist"></a></li>
+                    <li><a href=""><img src="./images/twitter.svg" alt="twitterlogo"></a></li>
+                    <li><a href=""><img src="./images/medium.svg" alt="mediumlogo"></a></li>
+                </ul>
+            </div>
         </div>
+        <div class="programmingcontainer">
+            <div class="programmingcontainer-row1">
+                <div class="storysection">
+                    <h2 class="storytitle">My Recent Work</h2>
+                    <img class="storyimage-title" src="./images/Desktop/story-page/Title.svg" alt="">
+                    <div class="storysection-columns">    
+                        <div class="storysection-column1">
+                            <ul class="storyimages">
+                                <li class="storyimages-underline"><img class="underline" src="./images/story/Vector.png" alt=""></li>
+                                <li><img class="secondmain" src="./images/SecondPage/ImgPlaceholder.png" alt=""></li>
+                            </ul>
+                        </div>
+                        <div class="storysection-column2" >
+                            <h3 class="storysubtitle">Multi-Post Stories</h3>
+                            <p class="storyparragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                            <ul class="storyimages2">
+                                <li class="img1"><a href=""><img class="skill-image-tag1" src="./images/SecondPage/Tag.png" alt="css"></a></li>
+                                <li class="img2"><a href=""><img class="skill-image-tag1" src="./images/SecondPage/Tag1.png" alt=""></a></li>
+                                <li class="img3"><a href=""><img class="skill-image-tag1" src="./images/SecondPage/Tag2.png" alt=""></a></li>
+                                <li class="img4"><a href=""><img class="skill-image-tag1" src="./images/SecondPage/Tag3.png" alt=""></a></li>
+                            </ul>
+                            <p><button class="storybotton" type="submit">See Project</button></p>
+                        </div>
+                    </div>
+                </div>
+            </div>
```

Comment:
- [ ] Kindly remove the line that is above the image to improve your Ui design.
- [ ] Please also as required in the project requirements, you must stick to the design as much as possible (margins top, bottom, right, and left, font and font sizes) using the Figma Template provided.
- [ ] Change the `font-family` of the project description to the font that is provided in the Figma design.
- [ ]  Let's make your button design better by removing the default border. You can use `border: none` for that.
- Your design:
<img width="769" alt="image" src="https://user-images.githubusercontent.com/77190203/154493747-7e60e682-0427-4174-8788-f7b8d3876b6f.png">

- Figma design:
<img width="555" alt="image" src="https://user-images.githubusercontent.com/77190203/154494199-76fe993c-a474-4bd2-af7e-ea7df0fc1f4f.png">